portfolio/src/main/webapp/script.js

```diff
@@ -16,9 +16,117 @@
  * Creates a map and adds it to the page
  */
 function initMap() {
-  const map = new google.maps.Map(
-    document.getElementById("map"),
-    {center: { lat: 49.250, lng: -122.982 }, zoom: 12});
+  const styledMapType = new google.maps.StyledMapType(
+    [
+      {
+        "featureType": "landscape.man_made",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#ffe4e1"
+          }
+        ]
+      },
+      {
+        "featureType": "landscape.natural",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#d5e9dd"
+          }
+        ]
+      },
+      {
+        "featureType": "poi.park",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#c5e5e0"
+          }
+        ]
+      },
+      {
+        "featureType": "road.highway",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#c3dbf2"
+          }
+        ]
+      },
+      {
+        "featureType": "road.highway",
+        "elementType": "geometry.stroke",
+        "stylers": [
+          {
+            "color": "#778899"
+          }
+        ]
+      },
+      {
+        "featureType": "water",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#d3f0fa"
+          }
+        ]
+      }
+    ],
+  {name: 'Styled Map'});
```

Comment:
NIT: Maybe store this big list as a separate mapStyle.json file and read from that local file when you need it? I think `parse()` should work nicely once you load the .json file but I haven't found a nice function to load local .json files.

Reply:
ah I wanted to do this too and I found this https://codepen.io/KryptoniteDove/post/load-json-file-locally-using-pure-javascript but it does not look pretty haha. We can both try it!

Reply:
Personally I think it's simpler to keep this in JS, but if you like the separation you can always move it to a separate map_styles.js file.

Reply:
ah ok, I was planning to separate the files anyways in a later PR, this file is getting way too long